project_reach/analysis_file.py

```diff
@@ -47,13 +47,45 @@ def generate(user, data, csv_by_message_output_path, csv_by_individual_output_pa
         avf_consent_withdrawn_key = "withdrawn_consent"
 
         # Translate keys to final values for analysis
-        show_keys = set()
-        for td in data:
-            AnalysisKeys.set_analysis_keys(user, td)
-            AnalysisKeys.set_matrix_keys(
-                user, td, show_keys, "S07E01_Humanitarian_Priorities (Text) - esc4jmcna_activation_coded",
-                "humanitarian_priorities"
-            )
+        show_keys = set()  # of all radio show matrix keys
+        AnalysisKeys.set_analysis_keys(user, data, {
```

Comment:
Why do we need this at all? It seems like what we're doing is creating a new name for variables that were in the coding scheme, for the purpose of prettifying the data format for analysis? Unless I'm misunderstanding, that seems like something better fixed procedurally?

Reply:
We need this on REACH because the names requested in the analysis file do not match those used by the pipeline. Better planning on future projects would allow us to move this translation earlier; re-engineering of RapidProTools and the flows in RapidPro itself is likely needed to be able to use the same keys everywhere.